### PR TITLE
[ALLUXIO-2666] Make BufferUtils Test Final

### PR DESCRIPTION
**Pull request by Doris11455**
https://alluxio.atlassian.net/browse/ALLUXIO-2666

### CODE REVIEW

**Comment by gpang** [comment]
alluxio-bot, check this please

File: core/common/src/test/java/alluxio/util/io/BufferUtilsTest.java [hunk 1]
```diff
@@ -22,8 +22,8 @@
 /**
  * Tests the {@link BufferUtils} class.
  */
-public class BufferUtilsTest {
-
+public final class BufferUtilsTest {
+/*public class BufferUtilsTest {*/
```
Comment (gpang) [inline core/common/src/test/java/alluxio/util/io/BufferUtilsTest.java]: This line should be removed.

**Comment by gpang** [comment]
Hi @Doris11455 . Thanks for this PR. Do you have any updates to your change?

**Comment by yupeng9** [comment]
@Doris11455 Feel free to reopen this PR once you have more updates.

**Pull request closed without merging**